README.md

```diff
@@ -69,25 +64,25 @@ const contentHash = ipfs.add(htmlBundle, { pin: true })
 const articleInfo = {
   contentHash,
   author: {
-    name: "test-user",
-    url: "user-home-page",
-    description: "this is a test user",
+    name: 'test-user',
+    url: 'user-home-page',
+    description: 'this is a test user',
   },
-  description: "This is a piece of test content",
-  image: "image-url",
+  description: 'This is a piece of test content',
+  image: 'image-url',
 }
 
 // this create the standard format of meta data,
 // should be merged with ISCN standard in the future
 const metaData = makeMetaData(articleInfo)
 
 const cid = await ipfs.dag.put(metaData, {
-  format: "dag-cbor",
+  format: 'dag-cbor',
   pin: true,
-  hashAlg: "sha2-256",
+  hashAlg: 'sha2-256',
 })
 // this is the final media hash used in the end of article url at matters.news
-const mediaHash = cid.toV1().toString()	// cid.toBaseEncodedString()
+const mediaHash = cid.toV1().toString() // cid.toBaseEncodedString()
```

Comment:
we'll probably remove all this section related to `makeMetaData`, it's not useful and very little people know how to use and what's it for